packages/widgets/src/components/CollapsibleNavBar/CollapsibleNavBar.tsx

```diff
@@ -0,0 +1,153 @@
+import { useLocation } from 'react-router-dom';
+import * as S from './styles';
+
+import {
+  Accordion,
+  AccordionGroup,
+  AccordionGroupContextProvider,
+  AccordionTitle,
+  ResponsiveImage,
+  Wrapper,
+} from '@ssa-ui-kit/core';
+
+import CollapsibleNavBarBase from './CollapsibleNavBarBase';
+import CollapsibleNavBarWrapper from './CollapsibleNavBarWrapper';
+import CollapsibleNavBarList from './CollapsibleNavBarList';
+import CollapsibleNavBarItem from './CollapsibleNavBarItem';
+import CollapsibleNavBarLink from './CollapsibleNavBarLink';
+import CollapsibleNavToggle from './CollapsibleNavToggle';
+
+import { INavBarExtendedProps } from './types';
+import { NavBarAccordionContent } from './NavBarAccordionContent';
+import { CollapsibleNavBarPopover } from './NavBarPopover';
+import { NavContentToggle } from './CollapsibleNavContentToggle';
+import { TriggerIcon } from './TriggerIcon';
+
+/**
+ * UI Component that shows the collapsible navigation bar
+ *
+ */
+export const CollapsibleNavBar = ({ items }: INavBarExtendedProps) => {
+  const { pathname } = useLocation();
+
+  return (
+    <CollapsibleNavBarBase>
+      <CollapsibleNavToggle />
+
+      <CollapsibleNavBarWrapper>
+        <Wrapper css={S.LogoWrapper}>
+          <ResponsiveImage
```

Comment:
This should go from props. The logo shouldn't be hardcoded: the library can be used by the community, not only by us.